### DISCUSSION TITLE
getting `React is not defined` on brand new project

### DISCUSSION THREAD

**Discussion by chrisdrackett** [vercel/next.js#17146] [Help]
# Bug report

## Describe the bug

I just started a new next.js project using `yarn create next-app` I then ran `yarn dev` within the generated folder and immediately get the error: `ReferenceError: React is not defined`

## To Reproduce

- run `yarn create next-app`
- choose a directory
- within the above project run `yarn dev`

## Expected behavior

The app runs without error

## System information

- OS: macOS
- Browser (if applies) Safari
- Version of Next.js: next@9.5.2
- Version of Node.js: 12.14.0

**Comment by vitaliemiron** [comment]
try to update nodejs

**Comment by justAdevTV** [comment]
@vitaliemiron I don't think it's the node version.

> Node.js 10.13 or later

See from here: https://nextjs.org/docs#system-requirements

**Comment by justAdevTV** [comment]
What does your `package.json` look like?

**Comment by timneutkens** [comment]
Converted to a github discussion given this is not reproducible.

**Comment by chrisdrackett** [comment]
I think this *might* have had to do with me having a `next.config.js` or `package.json` above this project folder (I created the project within another repo). I'm not entirely sure, but it seems that doing this outside of the folder I did it in originally fixed the issue.
